addons/event_stand/models/event_stand_slot_registration.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStandSlotRegistration(models.Model):
+    _name = 'event.stand.slot.registration'
```

Comment:
Why not shortening to event.stand/booth.registration ? Booking a stand is taking some slots from a given stand. It could ease reading without impacting too much understanding of models.